lib/datapacksjob.js

```diff
@@ -2396,6 +2400,13 @@ DataPacksJob.prototype.deployJob = async function(jobInfo) {
             VlocityUtils.verbose('Getting DataPacks', fullDataPath, tempJobInfo.manifest);
     
             let dataJSON = await this.vlocity.datapacksbuilder.buildImport(fullDataPath, tempJobInfo);
+            
```

Comment:
I think if you want this to fully apply to deploy you will also have to run the updateDataPackWithNewMappings from as part of the deployment itself as well. This would be great to add. buildImport would be a good possible place to put it. But may require other changes (hopefully not though)